brainscore_language/models/lm1b/__init__.py

```diff
@@ -0,0 +1,308 @@
+import functools
+import logging
+import urllib.request
+from typing import Tuple, Union, List, Dict
+
+import os
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3' # Disable verbose TF C++ output
```

Comment:
```suggestion
os.environ['TF_CPP_MIN_LOG_LEVEL'] = os.getenv('TF_CPP_MIN_LOG_LEVEL', '3')  # Disable verbose TF C++ output
```
Allow user to set variable manually without overwriting it